sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 5.0.0-beta.3 (Unreleased)
+## 5.0.0-beta.3 (2021-05-13)
 
+### Key Bug Fixes
```

Comment:
Should update to mention the breaking change that was just merged in https://github.com/Azure/azure-sdk-for-net/pull/21054

Reply:
MIght not be a bad idea.